addons/mail/models/mail_message.py

```diff
@@ -141,6 +141,9 @@ def default_get(self, fields):
     author_avatar = fields.Binary("Author's avatar", related='author_id.avatar_128', depends=['author_id'], readonly=False)
     author_guest_id = fields.Many2one(string="Guest", comodel_name='mail.guest')
     is_current_user_or_guest_author = fields.Boolean(compute='_compute_is_current_user_or_guest_author')
+    is_current_user_can_unlink = fields.Boolean(
```

Comment:
I'm not sure to see the point of this field.

Reply:
from the spec: admin can edit or delete a message from a channel